src/arrayMethodSort.js

```diff
@@ -5,8 +5,23 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
```

Comment:
it would be a really good idea to introduce a default sorting function
and use it as a default variable
```suggestion
  [].__proto__.sort2 = function(compareFunction = defaultCompareFunction) {
```